### DISCUSSION TITLE
Regd Xcode

### DISCUSSION THREAD

**Discussion by anjanaram1** [biostars/biostar-handbook-issues#290] [Q&A] [answered]
Hello - First off thanks to the author for providing clear and helpful responses. Xcode is taking up a lot of space on my Mac and causing other programs to not run or crash. Is it possible to continue with the installation process without xcode?

**Comment by ialbert** [comment]
try using just:

```
xcode-select --install
```

from command line, 

and perhaps that will suffice

**Reply by anjanaram1** [reply]
It says it is already installed.

**Reply by ialbert** [reply, marked as answer]
then should be all good
